src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -16,6 +16,68 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests
 {
     public class SyntaxNormalizerTests
     {
+        [Fact, WorkItem(52543, "https://github.com/dotnet/roslyn/issues/52543")]
+        public void TestNormalizeSwitchExpression()
+        {
+            TestNormalizeStatement(
+                @"var x = (int)1 switch { 1 => ""one"", 2 => ""two"", 3 => ""three"", {} => "">= 4"" };",
+                @"var x = (int)1 switch
+{
+  1 => ""one"",
+  2 => ""two"",
+  3 => ""three"",
+  {} => "">= 4""
+};"
+            );
+        }
+
+        [Fact, WorkItem(52543, "https://github.com/dotnet/roslyn/issues/52543")]
+        public void TestNormalizeSwitchExpressionComplex()
+        {
+            var a = @"var x = vehicle switch
+            {
+                Car { Passengers: 0 } => 2.00m + 0.50m,
+                Car { Passengers: 1 } => 2.0m,
+                Car { Passengers: 2 } => 2.0m - 0.50m,
+                Car c => 2.00m - 1.0m,
+
+                Taxi { Fares: 0 } => 3.50m + 1.00m,
+                Taxi { Fares: 1 } => 3.50m,
+                Taxi { Fares: 2 } => 3.50m - 0.50m,
+                Taxi t => 3.50m - 1.00m,
+
+                Bus b when ((double)b.Riders / (double)b.Capacity) < 0.50 => 5.00m + 2.00m,
+                Bus b when ((double)b.Riders / (double)b.Capacity) > 0.90 => 5.00m - 1.00m,
+                Bus b => 5.00m,
+
+                DeliveryTruck t when (t.GrossWeightClass > 5000) => 10.00m + 5.00m,
+                DeliveryTruck t when (t.GrossWeightClass < 3000) => 10.00m - 2.00m,
+                DeliveryTruck t => 10.00m,
+                { } => -1, //throw new ArgumentException(message: ""Not a known vehicle type"", paramName: nameof(vehicle)),
+                null => 0//throw new ArgumentNullException(nameof(vehicle))
+            };";
+            var b = @"var x = vehicle switch
+{
+  Car { Passengers: 0 } => 2.00m + 0.50m,
+  Car { Passengers: 1 } => 2.0m,
+  Car { Passengers: 2 } => 2.0m - 0.50m,
+  Car c => 2.00m - 1.0m,
+  Taxi { Fares: 0 } => 3.50m + 1.00m,
+  Taxi { Fares: 1 } => 3.50m,
+  Taxi { Fares: 2 } => 3.50m - 0.50m,
+  Taxi t => 3.50m - 1.00m,
+  Bus b when ((double)b.Riders / (double)b.Capacity) < 0.50 => 5.00m + 2.00m,
+  Bus b when ((double)b.Riders / (double)b.Capacity) > 0.90 => 5.00m - 1.00m,
+  Bus b => 5.00m,
+  DeliveryTruck t when (t.GrossWeightClass > 5000) => 10.00m + 5.00m,
+  DeliveryTruck t when (t.GrossWeightClass < 3000) => 10.00m - 2.00m,
+  DeliveryTruck t => 10.00m,
+  {} => -1, //throw new ArgumentException(message: ""Not a known vehicle type"", paramName: nameof(vehicle)),
+  null => 0 //throw new ArgumentNullException(nameof(vehicle))
+};";
+            TestNormalizeStatement(a, b);
```

Comment:
is thre a test that shows where the braces go if the parent is indented?

Reply:
Yes, always new line - like for if, while...